src/packetQuery/components/PacketQueryView.vue

```diff
@@ -213,10 +206,10 @@ export default {
         const bounds = this.openmct.time.bounds();
         const start = bounds.start;
         const end = bounds.end;
-        const timeSystem = openmct.time.timeSystem();
+        const timeSystem = this.openmct.time.timeSystem();
```

Comment:
I've created an issue to update to the latest time api methods: https://github.com/NASA-AMMOS/openmct-mcws/issues/284